### PR TITLE
llvm: switch to static

### PR DESCRIPTION
**Pull request by Andarwinux**

### CODE REVIEW

**Comment by shinchiro** [comment]
No lto for toolchain for whatever reason it is

**Comment by shinchiro** [comment]
Statically link mean the linker need extra time to produce final build.

File: toolchain/llvm/llvm-wrapper.cmake [hunk 1]
```diff
@@ -7,7 +7,6 @@ ExternalProject_Add(llvm-wrapper
     BUILD_COMMAND ""
     COMMAND ${CMAKE_COMMAND} -E make_directory ${MINGW_INSTALL_PREFIX}
     COMMAND ${CMAKE_COMMAND} -E make_directory ${MINGW_INSTALL_PREFIX}/lib
-    COMMAND ${CMAKE_COMMAND} -E create_symlink ${CMAKE_INSTALL_PREFIX}/bin/llvm-as        ${CMAKE_INSTALL_PREFIX}/bin/${TARGET_ARCH}-as
```
Comment (shinchiro) [inline toolchain/llvm/llvm-wrapper.cmake:10]: why?

File: toolchain/llvm/llvm-wrapper.cmake [hunk 2]
```diff
@@ -27,7 +26,7 @@ ExternalProject_Add(llvm-wrapper
     COMMENT "Setting up target directories and symlinks"
 )
 
-foreach(compiler clang++ g++ c++ clang gcc)
+foreach(compiler clang++ g++ c++ clang gcc as)
```
Comment (shinchiro) [inline toolchain/llvm/llvm-wrapper.cmake:29]: ..why added here when removed before?

**Comment by shinchiro** [comment]
On my local 4-cpu local machine, this completed in 1 hour 20 minutes without LTO which is improvement. I'm not gonna turn on lto for toolchain since that is unnecessary

File: toolchain/llvm/llvm-wrapper.cmake (hunk 1 above)
Comment (Andarwinux) [reply toolchain/llvm/llvm-wrapper.cmake:10]: llvm-as is an LLVM development tool, not a GAS port of LLVM. clang provides an integrated assembler, users should use it.
This fixes aarch64 asm problem. x86 normally uses nasm, so this error will not be visible.

File: toolchain/llvm/llvm-wrapper.cmake (hunk 2 above)
Comment (Andarwinux) [reply toolchain/llvm/llvm-wrapper.cmake:29]: llvm-as is an LLVM development tool, not a GAS port of LLVM. clang provides an integrated assembler, users should use it.

**Pull request closed without merging**